lib/kitchen/directions/bake_numbered_exercise/v1.rb

```diff
@@ -61,6 +78,18 @@ def bake_solution(exercise:, number:, solution_stays_put:, divider: '. ')
         return
       end
 
+      if in_appendix
+        solution.id = "#{exercise.id}-solution"
```

Comment:
are lines 82-83 redundant with lines 93-94?

Reply:
or, could lines 93-94 go above the `in_appendix` if-statement since they happen in both cases?